src/code/z_player_lib.c

```diff
@@ -1304,35 +1351,45 @@ void func_80090D20(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s*
 
             gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_player_lib.c", 2804),
                       G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
-            gSPDisplayList(POLY_XLU_DISP++, stringData->dList);
+            gSPDisplayList(POLY_XLU_DISP++, bowStringInfoPtr->dList);
 
             Matrix_Pop();
 
             CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_player_lib.c", 2809);
         } else if ((this->actor.scale.y >= 0.0f) && (this->rightHandType == 10)) {
+            static Vec3f shieldQuadVertices[] = {
+                { -4500.0f, -3000.0f, -600.0f },
+                { 1500.0f, -3000.0f, -600.0f },
+                { -4500.0f, 3000.0f, -600.0f },
+                { 1500.0f, 3000.0f, -600.0f },
+            };
+
             Matrix_Get(&this->shieldMf);
-            func_80090604(globalCtx, this, &this->shieldQuad, D_80126154);
+            Player_SetShieldCollision(globalCtx, this, &this->shieldQuad, shieldQuadVertices);
         }
 
         if (this->actor.scale.y >= 0.0f) {
             if ((this->heldItemActionParam == PLAYER_AP_HOOKSHOT) ||
                 (this->heldItemActionParam == PLAYER_AP_LONGSHOT)) {
-                Matrix_MultVec3f(&D_80126184, &this->unk_3C8);
+                static Vec3f hookshotHandPos = { 100.0f, 1500.0f, 0.0f };
```

Comment:
this name `hookshotHandPos` doesn't really provide any information, it's obvious this is hand (`PLAYER_LIMB_R_HAND`) and hookshot (`PLAYER_AP_HOOKSHOT`) related

I would revert to `D_80126184` until more documentation (eg on `unk_3C8`)

Reply:
why would i revert it? its the position of the hookshot itself (i.e. in hand), while the hook position is the position of the tip of the hookshot (the spike)